internal/converter/internal/staticconvert/testdata/sanitize.river

```diff
@@ -42,13 +42,13 @@ loki.relabel "logs_integrations_integrations_windows_exporter_application" {
 loki.source.windowsevent "logs_integrations_integrations_windows_exporter_application" {
 	eventlog_name          = "Application"
 	xpath_query            = ""
-	bookmark_path          = "C:\\grafana_test\\Grafana Agent\\bookmarks.xml"
 	poll_interval          = "0s"
 	use_incoming_timestamp = true
 	forward_to             = [loki.relabel.logs_integrations_integrations_windows_exporter_application.receiver]
 	labels                 = {
 		job = "integrations/windows_exporter",
 	}
+	legacy_bookmark_path = "C:\\grafana_test\\Grafana Agent\\bookmarks.xml"
```

Comment:
@erikbaranowski for some reason this did not like having anymore than one space after the name and the equals sign.

Reply:
specifically here for static mode is where it gets called: https://github.com/grafana/agent/blob/b827bad63156fad4033f1c2ab67d285086bd157f/internal/converter/internal/staticconvert/staticconvert.go#L59